### PR TITLE
Add a setting for whether to enable/disable volume deletion protection

### PR DESCRIPTION
**Pull request by Jille**
I want to manually enable it for important volumes, and make sure software never ever unsets it.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -29,6 +29,12 @@
       "settable": ["value"],
       "value": "ext4"
     },
+    {
+      "name": "use_protection",
+      "description": "whether to enable/disable protection on volumes managed by this plugin",
```
Comment (costela) [inline config.json]:
```suggestion
      "description": "whether to enable/disable delete protection on volumes managed by this plugin",
```

**Comment by Jille** [comment]
Thanks for the patch. Should I rebase and you'll merge it?

**Comment by costela** [comment]
ah yes, sorry, forgot to comment :sweat_smile:

**Comment by Jille** [comment]
np - thanks for the reviews :)

**Comment by costela** [comment]
Thanks again for the contribution! :+1:

**Pull request merged**